Gemfile

```diff
@@ -6,6 +6,8 @@ gem 'ruby-fifo', github: 'shurizzle/ruby-fifo'
 
 group :development do
   gem 'rspec'
+  gem 'coveralls'
+  gem 'simplecov'  
```

Comment:
@dmcnulla

> I did not realize that Gemfile and .gemspec served the same purpose. I will remove them from Gemfile. 
  gemspec server to declare valid dependencies for a gem specifically, while Gemfile is mostly just for a current installation locally. So rubygems will use only dependencies from gemspec.

> I use rubocop for stylechecker/linter. rspec handles the expectations (asserts) I use in cucumber. I will move those to gemspec as well. 

Yes but I haven't seen any rspec file in the repo.